src/qt/modaloverlay.cpp

```diff
@@ -122,6 +124,9 @@ void ModalOverlay::tipUpdate(int count, const QDateTime& blockDate, double nVeri
     // show the percentage done according to nVerificationProgress
     ui->percentageProgress->setText(QString::number(nVerificationProgress*100, 'f', 2)+"%");
     ui->progressBar->setValue(nVerificationProgress*100);
+ #if defined(Q_OS_WIN)
+   *sp_tray = nVerificationProgress*190;
```

Comment:
What's the 190?

Reply:
size of button on 4k i bet

Reply:
about QWinTaskbarProgress    something tells me you have reasons not to use it haha

Reply:
the button as in the application icon in the task bar? In what unit, pixels? And if so, will that width change if you for instance put your task bar on the left or on the right (where they look completely different and are as horizontally spacious as you want).

The only problem I can see with QWinTaskbarProgress is that is seems to be there only since Qt 5.2 and it's part of winextras.

Reply:
Also if button sizes are taken into consideration, then DPI scaling should also be considered

Reply:
it's never change You could put 1190 with np . If you put say 30 .... I bet you'd understand already

Reply:
yeah that's about hsize in pixels. 
@scravy about latter , trust me it's not only problem

Reply:
this square with bitcoin logo in taskbar called 'button' in related docs